project/RELEASE-CHECKLIST.md

```diff
@@ -24,22 +24,45 @@ git remote add $GITHUBUSER git@github.com:$GITHUBUSER/docker.git
 
 ### 1. Pull from master and create a release branch
 
+#### Major releases
```

Comment:
I'd move this title to after the statement about the format of the releases (x.y.z) since it applies to all releases.